### ISSUE TITLE
error: conflicting types for ‘dgesvd_’

### ISSUE THREAD

**Issue by yangarbiter** [ntucllab/libact#177]
It seems there is another issue for c-extension installation is using both OpenBLAS and LAPACKE

https://stackoverflow.com/questions/61945709/lapacke-h-file-not-found-error-when-installing-libact

**Comment by ariapoy** [comment]
Hi @yangarbiter,

I resolve this bug by following steps:

1. Check `/usr/include/lapack.h` file and `LAPACK_dgesvd()` function.

```=h
#define LAPACK_dgesvd LAPACK_GLOBAL(dgesvd,DGESVD)
void LAPACK_dgesvd(
     char const* jobu, char const* jobvt,
     lapack_int const* m, lapack_int const* n,
     double* A, lapack_int const* lda,
     double* S,
     double* U, lapack_int const* ldu,
     double* VT, lapack_int const* ldvt,
     double* work, lapack_int const* lwork,
     lapack_int* info );
```

2. Update `libact/query_strategies/src/variance_reduction/variance_reduction.c` w.r.t above arguments. 
From line 8 to 10 as [Code gist link](https://gist.github.com/ariapoy/cc326cfdb7c8e8a9ad431ce196370202).

3. Install it locally by `python setup.py install`. We could install but with some <span style="color:red">**red words**</span> from `zip_safe flat` to `libact.query_stragigies`.

```
...
zip_safe flag not set; analyzing archive contents...
libact.query_strategies.__pycache__._hintsvm.cpython-36: module references __file__
libact.query_strategies.__pycache__._variance_reduction.cpython-36: module references __file__
...
Using /home/lu/.local/lib/python3.6/site-packages
Finished processing dependencies for libact==0.1.6
```

4. I'm not sure how to test if *variance_reduction.py* works or not. But other functions work fine.

And here is my environment:

```shell
# Ubuntu Ver.
$ lsb_release -a
No LSB modules are available.
Distributor ID:	Ubuntu
Description:	Ubuntu 20.04.1 LTS
Release:	20.04
Codename:	focal

# Python Ver.
$ python
Python 3.6.12 |Anaconda, Inc.| (default, Sep  8 2020, 23:10:56)

# liblapack Ver.
$ apt-cache policy liblapack-dev
liblapack-dev:
  Installed: (none)
  Candidate: 3.9.0-1build1
  Version table:
     3.9.0-1build1 500
        500 http://archive.ubuntu.com/ubuntu focal/main amd64 Packages
```

Please check this solution works or not at your earliest convenience.
Suppose you need more information about my environment. Please let me know :smile: 

Sincerely.
Poy

**Comment by yangarbiter** [comment]
Hi Poy,

Thanks for reporting this!
I tested the change in libact/query_strategies/src/variance_reduction/variance_reduction.c and it worked!
Would you like to change the file with what you have in your gist link and submit a pull request?

Thanks.

**Comment by yangarbiter** [comment]
Resolved in #189
